### ISSUE TITLE
How to train a SVR(nb_primitives:1) from scratch?

### ISSUE THREAD

**Issue by sumai-1998** [ThibaultGROUEIX/AtlasNet#68]
You used a pretrained model in AtlasNet-master/training/trained_models/atlasnet_singleview_1_sphere called "log/atlasnet_singleview_1_sphere_tmp_True/network.pth", which has a "-5.5" train/val loss in 'Curves in log'.    
However if i train without a pretrained model, i just reach at '-5 train and -2 val' loss, which performs worse in 'display' and some airplane, and lack details in the other calsses.

**Comment by sumai-1998** [comment]
How to first train and get a log/atlasnet_singleview_1_sphere_tmp_True/network.pth?      ：-）

**Comment by sumai-1998** [comment]
This is mine train from scratch with the default settings, in SVR_1
![Default curve](https://github.com/4ssass1n0r/4ssass1n0r/blob/main/default.jpg)
This is your in training/trained_models/atlasnet_singleview_1_sphere
![Authors curve](https://github.com/4ssass1n0r/4ssass1n0r/blob/main/atlasnet_SVR_1_sphere.jpg)

**Comment by ThibaultGROUEIX** [comment]
Hi @4ssass1n0r,
Single-view networks are trained in 3 steps:
1- Train the atlasnet decoder in an autoencoder setting (with a pointnet encoder)
2- train the resnet encoder with the freezed atlasnet decoder
3- fine-tune all weights of the SVR (resnet+atlasnet)
You can find more details in the paper.
Best regards,
Thibault
